cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -1,9 +1,14 @@
 # -*- coding: utf-8 -*-
 """Test Service Details page functionality."""
+import time
+
 import pytest
 
+from markers.env_markers.provider import providers
+
```

Comment:
Why moving the import? It's local so it belongs to where it was originally.

Reply:
I think my flake8 was complaining and blocking commit in pre-commit hook hence I did that. But I have made it less stricter. I will update it now to original position.